Source/common/SNTXPCNotifierInterface.m

```diff
@@ -14,10 +14,16 @@
 
 #import "SNTXPCNotifierInterface.h"
 
+#import "SNTStoredEvent.h"
```

Comment:
On l24 `[SNTStoredEvent class]` is used to provide a list of classes in the collection sent over xpc.